inappmessaging/USERGUIDE.md

```diff
@@ -146,7 +146,11 @@ InAppMessaging.errorCallback = {
 }
 
 // Configure API: configure(context: Context): Boolean
+// In a Java app, this API is callable via `InAppMessaging.Companion.configure(context: Context)`.
 val iamFlag = InAppMessaging.configure(this)
+// Note that the subscription ID and config URL set from AndroidManifest.xml can also be overriden at runtime
+// using the extended configure API: configure (context: Context, subscriptionKey: String, configUrl: String).
+val iamFlag = InAppMessaging.configure(this, "change-to-your-subsrcription-key", "change-to-config-url")
```

Comment:
Can we also mention this method in `SDK Integration` section point 4?

Reply:
Added, please check it @imaginaris